juniorguru_chick/lib/threads.py

```diff
@@ -23,20 +24,41 @@ async def fetch_starting_message(thread: discord.Thread) -> discord.Message | No
     except discord.errors.NotFound:
         return None
 
+def name_thread(message: discord.Message, name_template) -> str | None:
+
+    brackets_regex = re.compile(r"""
+        ^\[     # starts with [
+        .*      # any number of any characters
+        [^\s]   # not a whitespace
+        [^\]]   # not a closing bracket
+        \]      # ends with ]
+        """, re.VERBOSE)
+
+    """If the message includes text in square brackets, use that as name for the thread. Otherwise, use the name template."""
+    if (match:= re.match(brackets_regex, message.content)) is not None:
+        content = match.group()[1:-1]
+        content = content.split(",")
+        processed_content = []
+        for word in content:
+            processed_content.append(word.lstrip().strip())
```

Comment:
```suggestion
            words.append(part.strip())
```

Having `lstrip().strip()` is redundant, so I removed the `lstrip()`.